src/Clock.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+
+class Clock extends React.Component {
+  constructor(props) {
+    super(props);
+    // Initialise component state to contain "date" attribute with current date and time
+    this.state = { date: new Date() };
+  }
+
+  componentDidMount() {
+    this.timerID = setInterval(() => this.tick(), 1000);
+  }
+
+  componentWillUnmount() {
+    clearInterval(this.timerID);
+  }
+
+  tick() {
+    this.setState({ date: new Date() });
+  }
+
+  render() {
+    return (
+      <div className="Clock">
+        {this.state.date.toLocaleString("en-GB", {
```

Comment:
It would be nice if we could also make the language dynamic and let the developer pass that as prop